metis-core/metis-core-rest/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:11-jre-slim
+COPY target/*.jar app.jar
```

Comment:
do we need an `apt run update` also to keep the container up to date?
This dependencies I remember are needed for media processing do we need them in this phase?
```
RUN apt install ffmpeg -y
RUN apt install ghostscript -y
RUN apt install imagemagick -y
```

Reply:
It doesn't have to be in fact, it's quite common since the containers are minimalistic so it should be fine.
The dependencies you are referring to are for the media library processing which is not performed in metis-core so those are not needed.

Reply:
oké :+1: